### PR TITLE
Updating steps to configure observability

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/11806#issuecomment-838524442

https://github.com/open-cluster-management/backlog/issues/10368

### CODE REVIEW

File: observability/observability_enable.adoc [hunk 1]
```diff
@@ -46,14 +50,30 @@ oc create secret generic multiclusterhub-operator-pull-secret \
...
+kind: Secret
+metadata:
+  name: thanos-object-storage
+type: s3
+config:
+  bucket: YOUR_ACM_BUCKET
+  endpoint: YOUR_ACM_ENDPOINT
+  insecure: true
+  access_key: YOUR_ACM_ACCESS_KEY
+  secret_key: YOUR_ACM_SECRET_KEY
```
Comment (songleo) [inline observability/observability_enable.adoc]: I suggest updating the config like the following:

```
    config:
      bucket: YOUR_S3_BUCKET
      endpoint: YOUR_S3_ENDPOINT
      insecure: true
      access_key: YOUR_ACCESS_KEY
      secret_key: YOUR_SECRET_KEY
```
`YOUR_ACM_XXX` maybe misleading to users.

File: observability/observability_enable.adoc [hunk 2]
```diff
@@ -113,7 +141,29 @@ config:
...
 ----
 +
-For more details, see https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.4/html/deploying_openshift_container_storage/deploying-openshift-container-storage-on-openshift-container-platform_rhocs#installing-openshift-container-storage-operator-using-the-operator-hub_aws-vmware[Installing Red Hat OpenShift Container Storage].
+For more details, see https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/deploying_openshift_container_storage/deploying-openshift-container-storage-on-openshift-container-platform_rhocs#installing-openshift-container-storage-operator-using-the-operator-hub_aws-vmware[Installing {ocp}].
+
+... You can retrieve the S3 access key for your cloud providers with the following commands:
+//1. Is this step needed? 2. How should the command be updated if it is needed?
++
+----
+ACCESS_KEY=$(oc -n <your-object-storage> get secret <oject-storage-secret> -o yaml | grep AccessKey | awk '{print $2}' | base64 --decode)
```
Comment (songleo) [inline observability/observability_enable.adoc]: `oject` -> `object`

File: observability/observability_enable.adoc [hunk 3]
```diff
@@ -113,7 +141,29 @@ config:
...
+
+echo $ACCESS_KEY
+----
+... You can retrieve the S3 secret key for your cloud providers with the following commands:
++
+----
+SECRET_KEY=$(oc -n <your-object-storage> get secret r<oject-storage-secret> -o yaml | grep SecretKey | awk '{print $2}' | base64 --decode)
```
Comment (songleo) [inline observability/observability_enable.adoc]: ditto

**Comment by dockerymick** [comment]
@songleo Thanks for your review and feedback!

**Comment by dockerymick** [comment]
Thanks @clyang82 for your review! Based on both of you all's review and my updates, I am comfortable with merging this PR

**Pull request merged**